recipes/cd3-boost-unit-definitions/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import (
+    apply_conandata_patches,
+    export_conandata_patches,
+    get,
+    copy,
+)
+from conan.tools.layout import cmake_layout
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "cd3-boost-unit-definitions"
+    description = "A collection of pre-defined types and unit instances for working with Boost.Units quantities."
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case not listed there, use "LicenseRef-<license-file-name>"
```

Comment:
Please clean up some of the comments from the template 🙏

Reply:
Sorry, I'll clean them all out.